music_assistant/providers/jellyfin/__init__.py

```diff
@@ -454,7 +456,15 @@ async def get_stream_details(
         jellyfin_track = await self._client.get_track(item_id)
         mimetype = self._media_mime_type(jellyfin_track)
         media_stream = jellyfin_track[ITEM_KEY_MEDIA_STREAMS][0]
-        url = self._client.audio_url(jellyfin_track[ITEM_KEY_ID], SUPPORTED_CONTAINER_FORMATS)
+        if jellyfin_track[ITEM_KEY_MEDIA_TYPE] == "Video":
```

Comment:
Right now the code on dev assumes there is a MediaStream[0] and if there is len(MediaStream) > 1, then the first one is the right one. This has been something I've wanted to change.

I think you looping over all streams is something we should do for all streams, regardless of the ITEM_KEY_MEDIA_TYPE.

Maybe loop once and find a supported codec of type audio, and just stream it as is. If that falls through, loop again and find video containers and transcode those.

If neither matches right now i think its best if we raise a MusicAssistantError.